### ISSUE TITLE
demos don't appear in Index on readthedocs

### ISSUE THREAD

**Issue by timpitman** [stellargraph/stellargraph#1506]
### Describe the bug

While our demos are now on readthedocs (and can be searched there), they don't appear in the index at https://stellargraph.readthedocs.io/en/latest/genindex.html

**Comment by timpitman** [comment]
Looking at the nbsphinx documentation: https://nbsphinx.readthedocs.io/en/0.6.1/usage.html I can't see a way to do this

**Comment by huonw** [comment]
Yeah, it looks like they don't have a very comprehensive index: https://nbsphinx.readthedocs.io/en/0.6.1/genindex.html
